### ISSUE TITLE
Create android build and upload to releases

### ISSUE THREAD

**Issue by avdept** [avdept/JellyBoxPlayer#35]

**Comment by ElijahHW** [comment]
Any chance you would be able to upload this to Fdroid repo as well? Quite a lot of people who use that as their only app store.

**Comment by avdept** [comment]
> Any chance you would be able to upload this to Fdroid repo as well? Quite a lot of people who use that as their only app store.

Sure, why not! I didn't use that marketplace, but how hard/long does it take to upload app there? Also any payments?

**Comment by ElijahHW** [comment]
It's mostly a store front for open-source and privacy focused applications. Publishing to Fdroid should be free of charge. Have never done it myself, nor am I a developer, so wouldn't know the complexity of it. But this seems to be the gist of it here: https://f-droid.org/docs/Submitting_to_F-Droid_Quick_Start_Guide/

**Comment by avdept** [comment]
Yeah it seems like it's really simple and straightforward. I definitely consider this as an option. I'll check if there's fast lane tools to push apps to fdroid store

**Comment by avdept** [comment]
@ElijahHW I just built APK and posted to releases. https://github.com/avdept/JellyBoxPlayer/releases/tag/v1.7.0

**Comment by Eschguy** [comment]
It's reporting at v0.1.7 instead of v1.7.0

**Comment by Eschguy** [comment]
> > It's reporting at v0.1.7 instead of v1.7.0
> 
> Apologize for messing with versions. Will get it correct in 1.8

Do you think you could do a 1.7.1 to correct it? That way obtainium doesn't constantly try to update it?

**Comment by avdept** [comment]
> > > It's reporting at v0.1.7 instead of v1.7.0
> > 
> > Apologize for messing with versions. Will get it correct in 1.8
> 
> Do you think you could do a 1.7.1 to correct it? That way obtainium doesn't constantly try to update it?

Sure. I'll do it first thing in a
Morning tomorrow

**Comment by avdept** [comment]
@Eschguy Made new build here - https://github.com/avdept/JellyBoxPlayer/releases/tag/v1.7.1
Please try it out and let me know if it works for you!

**Comment by Eschguy** [comment]
> @Eschguy Made new build here - https://github.com/avdept/JellyBoxPlayer/releases/tag/v1.7.1
> Please try it out and let me know if it works for you!

We're good! Thank you for doing that.
